tests/test_source.py

```diff
@@ -0,0 +1,56 @@
+"""Source tests"""
+from pyne.source import PointSource
+import os
+import warnings
+
+from unittest import TestCase
+import nose
+
+from nose.tools import assert_equal, assert_not_equal, assert_raises, raises, \
+    assert_almost_equal, assert_true, assert_false, assert_in
+
+from pyne.utils import QAWarning
+warnings.simplefilter("ignore", QAWarning)
+
+
+###############################################################################
+# tests the constructor
+def test_point_source():
+    pt_src = PointSource(1, 2, 3, 4, 5, 6, 12, "p", 0.5)
+    assert_equal(pt_src.x, 1)
+    assert_equal(pt_src.y, 2)
+    assert_equal(pt_src.z, 3)
+    assert_equal(pt_src.i, 4)
+    assert_equal(pt_src.j, 5)
+    assert_equal(pt_src.k, 6)
+    assert_equal(pt_src.particle, "p")
+    assert_equal(pt_src.weight, 0.5)
+
+
+################################################################################
+# test write particle for mcnp
+def test_point_source_mcnp():
```

Comment:
This test should be split into 5 different tests since it is testing 5 different cases.

Reply:
I wanted to keep them together as this should grow, with other kind of source, and other MC-softwares.

I fear if I split this to much, we will have billion of tests....
So I planned was to regroup the test per tested  method.
Don't have a. strong opinion there. so if you feel strongly about it, I'll fix it.

Reply:
I feel strongly that they should be split up into separate tests. It is not uncommon to test the same method/function in many different ways. Making separate tests will make it more clear what is failing.

If there is a concern about trying to keep track of all the tests when more source methods are added, then I think that can be resolved by separating them into different classes.

Reply:
will tackle this asap :)

Reply:
Maybe parametrize some of these? http://doc.pytest.org/en/latest/example/parametrize.html